docker/develop/x86_64/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04 as prep
+LABEL MAINTAINER="https://discord.gg/Uz29ny"
+
+RUN apt-get update && \
+    apt-get -y install curl xz-utils && \
+    apt-get autoclean -y && \
+    rm -rf /var/lib/apt/lists/*
+WORKDIR /
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN curl -L -o /stash $(curl -s https://api.github.com/repos/stashapp/stash/releases/tags/latest_develop | awk '/browser_download_url/ && /stash-linux/' | sed -e 's/.*: "\(.*\)"/\1/') && \
+    chmod +x /stash && \
+    curl --http1.1 -o /ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz && \
+    tar xf /ffmpeg.tar.xz && \
+    rm ffmpeg.tar.xz && \
+    mv /ffmpeg*/ /ffmpeg/
+
+FROM ubuntu:18.04 as app
+RUN apt-get update && \
+    apt-get -y install ca-certificates && \
+    adduser stash --gecos GECOS --shell /bin/bash --disabled-password --home /home/stash
+COPY --from=prep /stash /ffmpeg/ffmpeg /ffmpeg/ffprobe /usr/bin/
+EXPOSE 9999
```

Comment:
USER directive required to implement the additional user thats added.  Same issues will persist from previous comment.